### PR TITLE
Letters update

### PR DESCRIPTION
**Pull request by joshlow**
— add ref no to letters
— add text about adding ref no. to documents
— amended letter references in staff guides

@colinbruce note
Also addresses [ticket](https://www.pivotaltracker.com/story/show/124493221) to provide a page with all letters that staff can link to from the guide (at some point in the future)... this also makes it easier to debug changes to the letters!

### CODE REVIEW

File: config/locales/letters_en.yml [hunk 1]
```diff
@@ -104,52 +110,56 @@ en-GB:
...
-          <li>by post with a cheque written out to ‘HM Courts and Tribunals Service’ for the amount</li>
+          <li>by post with a cheque written out to ‘HM Courts and Tribunals Service’ for %{part_payment_amount}</li>
           <li>by debit card over the phone</li>
           <li>in person with a cheque, cash or debit card at the court counter</li>
         </ul>
-        <p>Please make sure you pay this amount by %{date_for_part_payment}</p>
+        <p>Please make sure you pay <strong>%{part_payment_amount}</strong> by <strong>%{date_for_part_payment}</strong>.</p>
         <p><strong>If you don’t agree with the amount you have to pay</strong></p>
-        <p>If you disagree with our decision about the amount you have to pay, you can appeal to the court or tribunal's Delivery Manager within 14 days of receiving this letter.</p>
+        <p>If you disagree with our decision about the amount you have to pay, you can appeal to %{name_delivery_manager}, Delivery Manager at %{court_tribunal_name}. You need to do this within 14 days of receiving this letter.</p>
```
Comment (colinbruce) [inline config/locales/letters_en.yml]: This was discussed with Leigh.  The delivery manager details are not available, we also cannot be sure of the specific court, as some appeals go to a regional delivery manager, not necessarily the same court as the current user.

File: config/locales/letters_en.yml [hunk 2]
```diff
@@ -4,52 +4,58 @@ en-GB:
...
       reject_over_savings_html: |
         <p>Dear %{full_name}</p>
+        <p><strong>Reference: %{reference}</strong></p>
         <p>We have received your application for help with fees. Unfortunately you’re not eligible and will have to pay the full fee. This is because you have more than the maximum amount of savings and investments allowed.
         We’ve calculated this as follows:</p>
         <p>Fee to pay:%{fee_amount}</p>
         <p>Maximum amount of savings and investments allowed: %{savings_threshold}</p>
         <p>Your savings and investments total: %{savings_total}</p>
         <p><strong>If you disagree with this decision</strong></p>
         <p>If you disagree with our decision, you can appeal to %{name_delivery_manager}, Delivery Manager at %{court_tribunal_name}. You need to do this by %{date_for_appeal}.</p>
```
Comment (colinbruce) [inline config/locales/letters_en.yml]: This letter is not currently displayed, but we'll need to remove the reference to delivery manager and court tribunal too.

File: config/locales/letters_en.yml [hunk 3]
```diff
@@ -4,52 +4,58 @@ en-GB:
...
       reject_over_income_html: |
         <p>Dear %{full_name}</p>
+        <p><strong>Reference: %{reference}</strong></p>
         <p>We have received your application for help with fees. Unfortunately you’re not eligible and will have to pay the full fee. This is because you have more than the maximum amount of income allowed.
         We’ve calculated this as follows:</p>
         <p>Fee to pay:%{fee_amount}</p>
         <p>Maximum amount of income allowed: %{income_threshold}</p>
         <p>Your income total: %{income_total}</p>
         <p><strong>If you disagree with this decision</strong></p>
         <p>If you disagree with our decision, you can appeal to %{name_delivery_manager}, Delivery Manager at %{court_tribunal_name}. You need to do this by %{date_for_appeal}.</p>
```
Comment (colinbruce) [inline config/locales/letters_en.yml]: This letter is not currently displayed, but we'll need to remove the reference to delivery manager and court tribunal too.

**Comment by novotnyjakub** [comment]
👍  great improvement!

**Pull request merged**